tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/external/iso_3166_status.txt

```diff
@@ -3,7 +3,7 @@ AD	;	Officially assigned	;	Andorre (l')
 AE	;	Officially assigned	;	United Arab Emirates (the)
 AF	;	Officially assigned	;	Afghanistan
 AG	;	Officially assigned	;	Antigua and Barbuda
-AI	;	Officially assigned	;	Anguilla
+AI	;	Formerly used	;	Anguilla
```

Comment:
Officially assigned per https://www.iso.org/obp/ui/#iso:code:3166:AI so this is another mistake?

Reply:
"Formerly used" for "French Afars and Issas"; still "Officially assigned" for Anguilla.

@macchiati @srl295 It looks like all my changes to this file are due to my naively following the list of "Formerly used" codes shown by the ISO site, not knowing that the same code may be in different categories when was formerly used for one thing and is currently used for something completely different. So, I'll revert the whole file, OK?